src/Ticket.php

```diff
@@ -298,6 +298,25 @@ public function canMassiveAction($action, $field, $value)
                             return false;
                         }
                         break;
+                    case 'itilcategories_id':
+                        $cat = new ITILCategory();
+                        if ($cat->getFromDB($value)) {
+                            switch ($this->fields['type']) {
+                                case self::INCIDENT_TYPE:
+                                    if (!$cat->getField('is_incident')) {
```

Comment:
`CommonDBTM::getField()` is made to be used to display value. We cannot deprecate it as it is widely used, but we should not use it anymore.
```suggestion
                                    if (!$cat->fields['is_incident']) {
```

Reply:
`fields` does not work:

```
Uncaught Exception Error: Call to undefined method ITILCategory::fields() in /home/teclib/Dev/GLPI/master/src/Ticket.php at line 306
```